app/src/main/res/layout/fragment_map_scooter_item.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/scooterCV"
+    android:layout_width="300dp"
+    android:layout_height="350dp"
+    android:clickable="true"
+    android:visibility="visible"
+    app:cardCornerRadius="28dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/bg_cardview">
+
+        <TextView
+            android:id="@+id/scooterTV"
+            style="@style/Body1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/textView_margin_top"
+            android:layout_marginRight="@dimen/textView_secondary_margin_end"
```

Comment:
Don't use view type specific names in res names. Unless it's something like a "button" which is a bit more generic